definitions/npm/bcrypt_v0.8.x/flow_>=v0.29.x/bcrypt_v0.8.x.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+
+declare module bcrypt {
+  declare function genSaltSync(rounds?: number): string;
+  declare function genSalt(rounds: number, callback: (err: Error, salt: string) => void): void;
+  declare function genSalt(callback: (err: Error, salt:string) => void): void;
+  declare function hashSync(data: any, salt: string): string;
+  declare function hashSync(data: any, rounds: number): string;
+  declare function hash(data: any, salt: string, callback: (err: Error, encrypted: string) => void): void;
```

Comment:
Thank you for your advice!
I confirmed what the data type is. The `data` must be a string.
```js
const salt = bcrypt.genSaltSync();

bcrypt.hash('yamadayuki', salt, (err, encrypted) => {
  if (err) throw err;     // no error
  console.log(encrypted); // outputs encrypted string
});

bcrypt.hash(197, salt, (err, encrypted) => {
  if (err) throw err;     // throw error
  console.log(encrypted); // nope
});

> Error: data must be a string and salt must either be a salt string or a number of rounds

```

I'll fix it 🚀